docs/source/index.rst

```diff
@@ -114,18 +114,18 @@ Please see our `contributing guidelines <https://github.com/cleanlab/cleanlab/bl
    :hidden:
    :caption: Tutorials
 
-   tutorials/indepth_overview
-   tutorials/image
-   tutorials/text
-   tutorials/tabular
-   tutorials/audio
-   tutorials/dataset_health
-   tutorials/outliers
-   tutorials/multiannotator
-   tutorials/multilabel_classification
-   tutorials/token_classification
-   tutorials/pred_probs_cross_val
-   tutorials/faq
+   Getting Started With Cleanlab <tutorials/indepth_overview>
```

Comment:
```suggestion
   Workflows of Data-Centric AI <tutorials/indepth_overview>
```

Reply:
not sure this is necessarily a canonical "getting started" tutorial, since it's quite long and eg. Audio Classification may be better starting point for somebody with audio data